Makefile

```diff
@@ -0,0 +1,42 @@
+KUTTL_VERSION=0.1.0
+KIND_VERSION=0.8.1
+KUBERNETES_VERSION ?= 1.17.5
```

Comment:
```suggestion
KUBERNETES_VERSION ?= 1.17.5
KUBECONFIG=kubeconfig
```

Reply:
I think we shouldn't assume the kubeconfig to be always present in relative path

Reply:
will comeback with another PR maybe when we decide to run it over konvoy